misc/brackets

```diff
@@ -0,0 +1 @@
+Subproject commit 963cbcb18da33c96b61009009f6fb11746f1ca11
```

Comment:
This is adobe brackets mode for factor syntax highlight. It's a submodule - link to my project at https://github.com/kerabromsmu/brackets-factor